tests/test_aggs.py

```diff
@@ -143,17 +162,17 @@ def test_buckets_to_dict():
     a.bucket("per_author", "terms", field="author.raw")
 
     assert {
-        "terms": {"field": "tags"},
-        "aggs": {"per_author": {"terms": {"field": "author.raw"}}},
-    } == a.to_dict()
+               "terms": {"field": "tags"},
```

Comment:
Please remove all the formatting that's not related to this PR?

Reply:
Sorry for the inconvience! It's fixed.